### DISCUSSION TITLE
KPN iptv not working after UDM-SE update to 3.2.9

### DISCUSSION THREAD

**Discussion by jmh78** [fabianishere/udm-iptv#302] [Q&A]
### What is your ISP? (if relevant)

KPN

### Diagnostic Information

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
50: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.233.57.197/22 brd 10.233.59.255 scope global iptv
       valid_lft forever preferred_lft forever
10.233.56.0/22 proto kernel scope link src 10.233.57.197
213.75.112.0/21 via 10.233.56.1 metric 250
=== Service Logs ===
Dec 31 00:14:22 UDM-SE udm-iptvd[3170]: Obtaining IP address for VLAN interface
Dec 31 00:14:22 UDM-SE udm-iptvd[3224]: udhcpc: started, v1.30.1
Dec 31 00:14:22 UDM-SE udm-iptvd[3224]: udhcpc: sending discover
Dec 31 00:14:23 UDM-SE udm-iptvd[3224]: udhcpc: sending select for 10.233.57.197
Dec 31 00:14:23 UDM-SE udm-iptvd[3224]: udhcpc: lease of 10.233.57.197 obtained, lease time 3600
Dec 31 00:14:23 UDM-SE udm-iptvd[3170]: NATing IPTV network ranges (if necessary)
Dec 31 00:14:23 UDM-SE udm-iptvd[3170]: Creating static routes (if necessary)
Dec 31 00:14:23 UDM-SE udm-iptvd[3170]: Setting up IGMP Proxy
Dec 31 00:14:23 UDM-SE udm-iptvd[3170]: Using improxy...
Dec 31 00:14:23 UDM-SE udm-iptvd[3170]: Starting IGMP Proxy

### Describe the Bug

After an update to OS v3.2.9 I am not able to get IPTV working again.
I have ran the script again, but without success.
I have uninstalled the udm-iptv, rebooted and again installed, without success.

Tried both IGMP 2 & 3

### Expected Behavior

Not sure what to do next? it worked for more than a year before this update.

**Comment by DrLapacho** [comment]
Same here with German-Telekom Provider...
I dont know what to do.


**Diagnostic Information**
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
27: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 84.145.154.15 peer 62.155.240.95/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.240.95 proto kernel scope link src 84.145.154.15
=== Service Logs ===
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@208]: if_mtu = 9216
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@214]: if_falgs = 4163
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@178]: ppp0's ipv4 address is 84.145.154.15
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@195]: if_index = 27
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@208]: if_mtu = 1492
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@214]: if_falgs = 4305
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: ERROR[main@619]: exiting.........
Jan 19 21:32:00 UDM-Pro IMPROXY[12335]: ERROR[main@619]: exiting.........
Jan 19 21:32:00 UDM-Pro systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Jan 19 21:32:00 UDM-Pro systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

**===ifconfig==**
root@UDM-Pro:~# ifconfig
br0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9216
        inet 192.168.2.1  netmask 255.255.255.0  broadcast 0.0.0.0
        inet6 fe80::e438:83ff:fe23:52fa  prefixlen 64  scopeid 0x20<link>
        ether e6:38:83:23:52:fa  txqueuelen 1000  (Ethernet)
        RX packets 28802  bytes 6851889 (6.5 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 80713  bytes 94754358 (90.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52f9  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:f9  txqueuelen 1000  (Ethernet)
        RX packets 12  bytes 1154 (1.1 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 20  bytes 3906 (3.8 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52fa  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:fa  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 3016 (2.9 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52fb  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:fb  txqueuelen 1000  (Ethernet)
        RX packets 7  bytes 338 (338.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 22  bytes 4126 (4.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52fc  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:fc  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18  bytes 3166 (3.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52fd  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:fd  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18  bytes 3166 (3.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52fe  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:fe  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18  bytes 3166 (3.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth6: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:52ff  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:52:ff  txqueuelen 1000  (Ethernet)
        RX packets 47  bytes 3608 (3.5 KiB)
        RX errors 0  dropped 3  overruns 0  frame 0
        TX packets 19  bytes 3252 (3.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth7: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:5300  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:53:00  txqueuelen 1000  (Ethernet)
        RX packets 37  bytes 3680 (3.5 KiB)
        RX errors 0  dropped 3  overruns 0  frame 0
        TX packets 20  bytes 3362 (3.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth8: flags=4099<UP,BROADCAST,MULTICAST>  mtu 9216
        inet6 fe80::e638:83ff:fe23:5301  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:53:01  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 3  bytes 304 (304.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth9: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:5302  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:53:02  txqueuelen 10000  (Ethernet)
        RX packets 69164  bytes 81130682 (77.3 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18032  bytes 5486086 (5.2 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth10: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 192.168.1.2  netmask 255.255.255.0  broadcast 0.0.0.0
        ether e4:38:83:23:53:03  txqueuelen 10000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth9.7: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e638:83ff:fe23:5302  prefixlen 64  scopeid 0x20<link>
        ether e4:38:83:23:53:02  txqueuelen 1000  (Ethernet)
        RX packets 69164  bytes 81130682 (77.3 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18034  bytes 5736219 (5.4 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 139285  bytes 41839826 (39.9 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 139285  bytes 41839826 (39.9 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

ppp0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST>  mtu 1492
        inet 84.145.154.15  netmask 255.255.255.255  destination 62.155.240.95
        ppp  txqueuelen 3  (Point-to-Point Protocol)
        RX packets 69013  bytes 80566065 (76.8 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17988  bytes 5338272 (5.0 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

switch0: flags=4419<UP,BROADCAST,RUNNING,PROMISC,MULTICAST>  mtu 9216
        inet6 fe80::e438:83ff:fe23:52fa  prefixlen 64  scopeid 0x20<link>
        ether e6:38:83:23:52:fa  txqueuelen 1000  (Ethernet)
        RX packets 29704  bytes 6906145 (6.5 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 80924  bytes 93666382 (89.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

switch0.1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9216
        inet6 fe80::e438:83ff:fe23:52fa  prefixlen 64  scopeid 0x20<link>
        ether e6:38:83:23:52:fa  txqueuelen 1000  (Ethernet)
        RX packets 28826  bytes 6853541 (6.5 MiB)
        RX errors 0  dropped 3  overruns 0  frame 0
        TX packets 80713  bytes 94754358 (90.3 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

**Describe the Bug**
After switsching from DSL to FTTH today Magenta TV stucks after 7-10 seconds.
All worked fine with DSL
UDM-Pro v3.2.9

![image](https://github.com/fabianishere/udm-iptv/assets/139555237/9b1d8376-a685-4812-95a1-d017f786abf8)
![image](https://github.com/fabianishere/udm-iptv/assets/139555237/1de0c1bf-8f87-41d1-8339-3170b5b72c40)
![image](https://github.com/fabianishere/udm-iptv/assets/139555237/c6cec251-1ec1-4964-884c-571e4b4846a4)

**udm-iptv.conf**

IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_VLAN="0"
IPTV_WAN_VLAN_INTERFACE="iptv"
IPTV_WAN_RANGES="224.0.0.0/4 87.141.0.0/16 193.158.0.0/15"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br0"
IPTV_IGMPPROXY_PROGRAM="improxy"
IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="false"
IPTV_IGMPPROXY_DEBUG="true"
IPTV_IGMPPROXY_IGMP_VERSION="3"


My mother-in-law can't watch TV, I suffer ;)
Thanks for help. 
Michael

**Reply by thomt44** [reply]
Disable igmp proxy and reinstall the config
